deepmd/pd/train/wrapper.py

```diff
@@ -64,74 +64,13 @@ def __init__(
-                        "Hybrid descriptor is not implemented yet"
-                    )
-                else:
-                    raise RuntimeError(f"Unknown class_type {class_type_base}!")
-                for link_item in shared_links[shared_item]["links"][1:]:
-                    class_type_link = link_item["shared_type"]
-                    model_key_link = link_item["model_key"]
-                    shared_level_link = int(link_item["shared_level"])
-                    assert (
-                        shared_level_link >= shared_level_base
-                    ), "The shared_links must be sorted by shared_level!"
-                    assert (
-                        "descriptor" in class_type_link
-                    ), f"Class type mismatched: {class_type_base} vs {class_type_link}!"
-                    if class_type_link == "descriptor":
-                        link_class = self.model[model_key_link].get_descriptor()
-                    elif "hybrid" in class_type_link:
-                        raise NotImplementedError(
-                            "Hybrid descriptor is not implemented yet"
-                        )
-                    else:
-                        raise RuntimeError(f"Unknown class_type {class_type_link}!")
-                    link_class.share_params(
-                        base_class, shared_level_link, resume=resume
-                    )
-                    log.warning(
-                        f"Shared params of {model_key_base}.{class_type_base} and {model_key_link}.{class_type_link}!"
-                    )
-            else:
-                if hasattr(self.model[model_key_base], class_type_base):
-                    base_class = self.model[model_key_base].__getattr__(class_type_base)
-                    for link_item in shared_links[shared_item]["links"][1:]:
-                        class_type_link = link_item["shared_type"]
-                        model_key_link = link_item["model_key"]
-                        shared_level_link = int(link_item["shared_level"])
-                        assert (
-                            shared_level_link >= shared_level_base
-                        ), "The shared_links must be sorted by shared_level!"
-                        assert (
-                            class_type_base == class_type_link
-                        ), f"Class type mismatched: {class_type_base} vs {class_type_link}!"
-                        link_class = self.model[model_key_link].__getattr__(
-                            class_type_link
-                        )
-                        link_class.share_params(
-                            base_class, shared_level_link, resume=resume
-                        )
-                        log.warning(
-                            f"Shared params of {model_key_base}.{class_type_base} and {model_key_link}.{class_type_link}!"
-                        )
+        raise NotImplementedError("share_params is not implemented yet")
```

Comment:
> Why is share_params not supported ?

I remove them for code coverage as this method seems not used currently in  water/se_e2_a/se_atten/DPA-2 with paddle backend.

Reply:
> Why is share_params not supported ?

I can re-add share_params if necessary...

Reply:
These are tested in source/tests/pt/test_multitask.py, maybe it's better to add this test in pd.

Reply:
> These are tested in `source/tests/pt/test_multitask.py`, maybe it's better to add this test in pd.

I think I might have mistakenly attributed another randomly occurring CI CUDA issue(#4439) to the `test_muktitask.py` file. I reviewed the file again to ensure it can correctly pass the unit tests.